### PR TITLE
Add Authme.

### LINKED ISSUE #790: add authme
- [x] I have checked that this app does not already have a pending request and has not previously been declined.
 - App Name:Authme
 - Download URL:https://github.com/Levminer/authme/releases/download/4.0.3/authme-4.0.3-linux-x64.deb
 - Website URL:https://github.com/Levminer/authme
 - App Summary:an authenticator app

### PR DESCRIPTION
**Pull request by marosg42**
Resolves #790

```
Authme
  Package:	authme
  Repository:	99-local
  Updater:	deb-get
  Installed:	4.0.3
  Published:	4.0.3
  Architecture:	amd64
  Download:	https://github.com/Levminer/authme/releases/download/4.0.3/authme-4.0.3-linux-x64.deb
  Website:	https://github.com/Levminer/authme
  Summary:	Simple cross-platform two-factor (2FA) authenticator app for desktop.
```

### CODE REVIEW

**Review by philclifford — APPROVED** [review]

**Comment by philclifford** [comment]
@marosg42 Oopsie - Pretty name and references here should say Authme not Authy (which is the Twilio one available via `snap` ) :confused: :bulb: .

File: 01-main/packages/authme [hunk 1]
```diff
@@ -0,0 +1,9 @@
+DEFVER=1
+get_github_releases "Levminer/authme" "latest"
+if [ "${ACTION}" != "prettylist" ]; then
+    URL="$(grep "browser_download_url.*\.deb\"" "${CACHE_FILE}" | head -n1 | cut -d'"' -f4)"
+    VERSION_PUBLISHED="$(echo "${URL}" | cut -d'-' -f2)"
+fi
+PRETTY_NAME="Authy"
```
Comment (philclifford) [inline 01-main/packages/authme:7]: PRETTY_NAME="Authme"

**Review by philclifford — CHANGES_REQUESTED** [review]
And ideally change the PR title and commit comment too ;-)

File: 01-main/packages/authme (hunk 1 above)
Comment (marosg42) [reply 01-main/packages/authme]: oopsie indeed. Thanks.

**Review by philclifford — APPROVED** [review]
Thank you!

**Pull request merged**